src/mimefactory.rs

```diff
@@ -541,8 +551,18 @@ impl<'a, 'b> MimeFactory<'a, 'b> {
             "Message-ID".into(),
             render_rfc724_mid(&rfc724_mid),
         ));
-
-        unprotected_headers.push(Header::new_with_value("To".into(), to).unwrap());
+        
+        if self.bcc_group {
+            // cs
+            // the app must not include Bcc header as MTA doesn't removes it!
+            // smtp receipients are independent from mail header !!
+            //unprotected_headers.push(Header::new_with_value("Bcc".into(), to).unwrap());
+            //
+            // this is what Thunderbird does when no "To:" header is there
+            unprotected_headers.push(Header::new("To".into(), "Hidden_receipients: ;".to_string()));
```

Comment:
Sometime ago, I heard that spam filters sometimes filter this out. The advice was to just put your own email address into `To`.

Reply:
Don't know what's the best.
I examined what Thunderbird does and did the same.